_partials/osx_oh_my_zsh.md

```diff
@@ -32,3 +32,27 @@ On Mac, open `Terminal > Preferences` and set the "Pro" theme as default in `Pro
 **Quit** and restart the Terminal. It should now have a nice black background, more easy on the eyes.
 
 :bulb: There are plenty of themes available on the Internet like [MaterialDark](https://github.com/lysyi3m/macos-terminal-themes#materialdark-download) if you fancy trying another one. That's something you can configure later during the day or come back to it if you are done with your setup early. Please carry on with the Github setup!
+
+## Apple Silicon computers
+
+<details>
+  <summary>Forgot if your computer uses Apple Silicon ?</summary>
```

Comment:
There is no space between word and punctuation in English 👍 Should be `Silicon?`

Reply:
Done 👍 🙏